### PR TITLE
draft for DAR 87

### PR DESCRIPTION
**Pull request by kelseiv**
This PR is on hold per request to Performance team to evaluate and verify whether the ≈20 GB of data on disk benchmark is valid: https://github.com/influxdata/influxdb/issues/17592

update per https://github.com/influxdata/DAR/issues/87

### CODE REVIEW

File: content/enterprise_influxdb/v1.7/administration/anti-entropy.md [hunk 1]
```diff
@@ -286,15 +286,14 @@ If a data node suddenly disappears due to a catastrophic hardware failure or for
...
 
-Perhaps you are replacing a machine that is being decommissioned, upgrading hardware, or something else entirely.
-The Anti-Entropy service will automatically copy shards to the new machines.
+To replace a machine running a data node, start the Anti-Entropy service to copy shards to the new machines by doing one of the following:
 
-Once you have successfully run the `influxd-ctl update-data` command, you are free
-to shut down the retired node without causing any interruption to the cluster.
-The Anti-Entropy process will continue copying the appropriate shards from the
-remaining replicas in the cluster.
+- For nodes with more than ≈20 GB of data on disk, ?
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.7/administration/anti-entropy.md:293]: what's our best practice here?

File: content/enterprise_influxdb/v1.7/guides/replacing-nodes.md [hunk 2]
```diff
@@ -274,7 +273,9 @@ The process of replacing data nodes is as follows:
 
 ### 2. Replace the old data node with the new data node
 
-Log into any of your cluster's meta nodes and use `influxd-ctl update-data` to replace the old data node with the new data node:
+- For nodes with more than ≈20 GB of data on disk, ?
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.7/guides/replacing-nodes.md:276]: ditto

**Comment by andrew-influx** [comment]
My recommendation of 20GB on disk for a datanode is a shot in the dark. Hate to drag this one out but would be good to get a dev resource to way in on it.

@dgnorton Perhaps you can weigh in on good recommendations of when to (or when not to )use the update-data command for a cluster? Mayhap it's more stable then I think?

**Comment by kelseiv** [comment]
Moved PR to [v2 repo](https://github.com/influxdata/docs-v2/pull/1304)

**Pull request closed without merging**